js/components/graph/Sidebar.js

```diff
@@ -1,204 +1,100 @@
-      this.setState({
-        focusDisabled: true,
-      });
-    }
-  }
-
-  createGraphButtons = () => {
-    return this.props.graphs.map((graph, i) => {
-      return (
-        <div
-          className="graph-button"
-          id={"graph-" + graph.id}
-          data-testid={"test-graph-" + i}
-          key={i}
-          onClick={() => this.props.updateGraph(graph.title)}
-        >
-          {graph.title}
-        </div>
-      )
-    });
-  }
-
-  createFocusButtons = () => {
-    const computerScienceFocusData = [
-      ["sci", "Scientific Computing"],
-      ["AI", "Artificial Intelligence"],
-      ["NLP", "Natural Language Processing"],
-      ["vision", "Computer Vision"],
-      ["systems", "Computer Systems"],
-      ["game", "Video Games"],
-      ["HCI", "Human Computer Interaction"],
-      ["theory", "Theory of Computation"],
-      ["web", "Web Technologies"],
-    ];
-
-    return computerScienceFocusData.map((focus, i) => {
-      const openDetails = this.props.currFocus == focus[0];
-      return (
-        <Focus
-          key={i}
-          pId={focus[0]}
-          data-testid={"test-focus-" + i}
-          focusName={focus[1]}
-          openDetails={openDetails}
-          highlightFocus={(id) => this.props.highlightFocus(id)}
-        />
-      )
-    });
+  getCurrentCourses = () => {
+    return <div>Test</div>
   }
 
   toggleSidebar = location => {
     if (this.state.toggled) {
       // close graph
       this.setState({
-        contentHidden: true,
-        graphActive: 1,
         toggled: false,
+        contentHidden: true,
       })
     } else if (!this.state.toggled && location === "button") {
       // open graph
       this.setState({
         toggled: true,
         contentHidden: false,
-        graphActive: 1,
-      });
-    }
-  }
-
-  showFocuses = focus => {
-    if (focus) {
-      // show focuses
-      this.setState({
-        graphActive: 0
-      });
-    } else {
-      // show graphs
-      this.setState({
-        graphActive: 1
       });
     }
   }
 
   // Sidebar rendering methods
-  renderSidebarHeader= () => {
-    const contentHiddenClass = this.state.contentHidden ? "hidden" : "";
+  renderFCE= () => {
     const fceString = Number.isInteger(this.props.fceCount) ? this.props.fceCount + ".0" : this.props.fceCount
 
     return (
-      <div id="fce" className={contentHiddenClass}>
+      <div id="fce">
         <div id="fcecount" data-testid="test-fcecount">FCE Count: {fceString}</div>
-        <button id="reset" data-testid="test-reset" onClick={() => this.props.reset()}>Reset Selection</button>
       </div>
     )
   }
 
-  renderSidebarNav = () => {
-    const focusDisabled = this.state.focusDisabled ? "disabled" : "";
-    const focusActiveClass = this.state.graphActive === 0 ? "active" : "";
-    const graphActiveClass = this.state.graphActive === 1 ? "active" : "";
+  renderSearchBar = () => {
+    const searchHidden = this.state.toggled ? "" : "hidden";
```

Comment:
You don't need this class here; it'll be enough to have the class only on the root sidebar component.